### PR TITLE
`qmk info`: nicer rendering of ISO enter

### PR DESCRIPTION
**Pull request by fauxpark**
## Description

image

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: lib/python/qmk/keyboard.py [hunk 1]
```diff
@@ -169,6 +169,7 @@ def render_layout(layout_data, render_ascii, key_labels=None):
         y = ceil(key.get('y', 0) * 3)
         w = ceil(key.get('w', 1) * 4)
         h = ceil(key.get('h', 1) * 3)
+        is_iso_key = w == 5 and h == 6
```
Comment (zvecr) [inline lib/python/qmk/keyboard.py:172]: Less dependent on the previous multiplication?
```suggestion
        is_iso_key = key.get('w', 1) == 1.25 and key.get('h', 1) == 2
```

However the above code is already a bit magic

Comment (fauxpark) [reply lib/python/qmk/keyboard.py:172]: That's what I originally had, but now we're doing the `.get()` twice. I figured the variable name was indication enough.

Ideally some of this code should be refactored out into "render a key" and possibly "render the ISO enter key" functions.

Comment (zvecr) [reply lib/python/qmk/keyboard.py:172]: I mention, as #15846 modifies this area and it wouldnt be addressed in a simple rebase.

**Comment by zvecr** [comment]
```console
$ qmk info -l -kb xd84
...
Layout Layout_75_Iso:
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│Es││F1││F2││F3││F4││F5││F6││F7││F8││F9││F1││F1││F1││Pr││Pa││De│
└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘
┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──────┐┌──┐
│~ ││1 ││2 ││3 ││4 ││5 ││6 ││7 ││8 ││9 ││0 ││_ ││+ ││Backsp││Ho│
└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──────┘└──┘
┌────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌────┐──┐
│Tab ││Q ││W ││E ││R ││T ││Y ││U ││I ││O ││P ││{ ││} ││Ente│Pa│
└────┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└┐   │──┘
┌─────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐│   │┌──┐
│Caps ││A ││S ││D ││F ││G ││H ││J ││K ││L ││: ││@ ││~ ││   ││Pa│
└─────┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└───┘└──┘
┌───┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌─────┐┌──┐┌──┐
│Shi││| ││Z ││X ││C ││V ││B ││N ││M ││< ││> ││? ││Shift││Up││En│
└───┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└─────┘└──┘└──┘
┌───┐┌───┐┌───┐┌───────────────────────┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│Ctr││Win││Alt││Space                  ││Al││Fn││Ct││Le││Do││Ri│
└───┘└───┘└───┘└───────────────────────┘└──┘└──┘└──┘└──┘└──┘└──┘
```
Renders fine on configurator.

File: lib/python/qmk/keyboard.py [hunk 2]
```diff
@@ -216,3 +199,64 @@ def render_layouts(info_json, render_ascii):
...
+
+    top_line = array('u', box_chars['tl'] + label_border_top + box_chars['tr'])
+    lab_line = array('u', box_chars['v'] + label_middle + box_chars['v'])
+    crn_line = array('u', box_chars['bl'] + box_chars['tr'] + label_blank + box_chars['v'])
+    mid_line = array('u', box_chars['v'] + label_blank + box_chars['v'])
+    bot_line = array('u', box_chars['bl'] + label_border_bottom + box_chars['br'])
+
+    textpad[y][x - 1:x + w + 1] = top_line
+    textpad[y + 1][x - 1:x + w + 1] = lab_line
+    textpad[y + 2][x - 1:x + w + 1] = crn_line
```
Comment (zvecr) [inline lib/python/qmk/keyboard.py:259]:
```suggestion
    textpad[y][x - 1:x + w] = top_line
    textpad[y + 1][x - 1:x + w] = lab_line
    textpad[y + 2][x - 1:x + w] = crn_line
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: lib/python/qmk/keyboard.py (hunk 2 above)
Comment (fauxpark) [reply lib/python/qmk/keyboard.py:259]: Nope, I think that's just left over from when I was trying to get it working and experimenting with offsets. All the lines should end at the same column.

**Review by zvecr — APPROVED** [review]

**Pull request merged**